test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Binders/FakeBindMethods.cs

```diff
@@ -40,6 +40,9 @@ internal class FakeBindMethods
         public static readonly SingleValueNode FakeSingleIntPrimitive =
             new ConstantNode(100);
 
+        public static readonly SingleValueNode FakeSingleSinglePrimitive =

```

Comment:
Even though a float is `EdmPrimitiveTypeKind.Single`, Maybe for a better description, you could have used `FakeSingleFloatPrimitive`

Reply:
I could do that